docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-solver-configuration-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='optimizer-solver-configuration-proc']
+= Create a Solver in {CENTRAL}
+
+The Solver editor creates a solver configuration that can be run in the Execution Solver or plain Java code after the KJAR is deployed.
+
+You can edit and create Solver configurations in {CENTRAL}.
+
+.Prerequisite
+{PRODUCT} has been downloaded and installed. Your project has been designed and deployed with all relevant assets configured. 
++ 
```

Comment:
This plus sign is rendered verbatim. I think it wasn't the intention.

Reply:
removed